vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
+**entando-k8s-service** is invoked by the ECM. 
 * GitHub: <https://github.com/entando-k8s/entando-k8s-service/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-k8s-service/>
 
-#### Customization
-It is very unlikely that the _entando-k8s-service_ will be customized as part of an Entando implementation.
-
-## keycloak
-The _entando-keycloak_ project is an extension of the base Keycloak images. The extension provides the default 
-themes for Entando, a customized realm and clients, and adds the Oracle ojdbc jars for connection to Oracle 
-databases.
+## Keycloak
+The **entando-keycloak** project is an extension of the base Keycloak images. It provides default themes for Entando, customized realm and clients, and Oracle JDBC jars for connecting to Oracle databases.
 * GitHub: <https://github.com/entando/entando-keycloak/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-keycloak/>
 
 #### Customization
-The keycloak image will often be customized as part of an Entando implementation. Common extensions will 
-include changing the theme, adding default connections, adding default social logins, adding default clients, 
-or other changes. 
+The Keycloak image can be customized as part of an Entando implementation. Some common extensions are: 
+* change the theme 
+* add default connections 
+* add default social logins 
+* add default clients 
 
 ## Other Key Repositories 
 ### entando-core
-The entando-core project is a J2EE application that exposes APIs for the Entando CMS, includes the legacy 
-admin console, and includes the portal-ui project that performs the server side composition for pages 
-rendered via an Entando application. Note that only the composition is performed server side. 
-Javascript code is rendered on the client. The entando-core is realized via an instance that includes the 
-WAR files generated from a core build as dependencies. In a default deployment this is the _entando-de-app_.
+The **entando-core** project is a J2EE application that exposes APIs for the Entando CMS, including the admin console and the portal UI project that performs the server side composition for pages rendered via an Entando application. Note that only the composition is performed server side. JavaScript code is rendered client-side. The **entando-core** is realized via an instance that includes the 
+WAR files generated from a core build as dependencies. In a default deployment this is the **entando-de-app**.
 
 * GitHub: <https://github.com/entando/entando-core/>
 * DockerHub: None (deployed to maven central)
 
 #### Customization
-For users familiar with versions prior to Entando 6, there will be cases where the _entando-core_ is customized. 
-In most cases, these customizations will be delivered via a WAR overlay in the instance project. 
+For users familiar with versions prior to Entando 6, there will be cases where the **entando-core** is customized. 
```

Comment:
sentence doesn't make sense - reads "users who are familiar will have to customize." rephrase so no faulty logic: "Users familiar with versions prior to Entando 6 know that there are cases where the **entando-core** must be customized."

Reply:
That changes the meaning and didn't want to do that. I don't think the sentence reads like it has to be customized, it says there will be cases